amf-api-contract/shared/src/main/scala/amf/apicontract/internal/validation/definitions/ParserSideValidations.scala

```diff
@@ -441,6 +458,12 @@ object ParserSideValidations extends Validations {
     ImplicitVersionParameterWithoutApiVersion.id -> all(WARNING), // TODO: should be violation
     InvalidVersionBaseUriParameterDefinition.id  -> all(WARNING), // TODO: should be violation
     HeaderMustBeObject.id                        -> Map(Async20Profile -> VIOLATION),
+    MissingAnnotationSchema.id -> Map(Raml10Profile -> VIOLATION,
```

Comment:
GraphQL ?

Reply:
Not for now, in fact graphql vendor is not in this branch (I added at another branch with graphql changes).